static/docs/get-started/reproduce.md

```diff
@@ -1,11 +1,12 @@
 # Reproduce
 
 In the previous chapters, we described our first
-[pipeline]](/doc/commands-reference/pipeline). Basically, we created a number of
-[stage files](/doc/commands-reference/run). Each of these
-[DVC-files](/doc/user-guide/dvc-file-format) describes single stage we need to
-run towards a final result (a pipeline). Each depends on some data (either raw
-data files or intermediate results from previous stages) and code files.
+[pipeline]](/doc/commands-reference/pipeline). Basically, we generated a number
+of [stage files](/doc/commands-reference/run)
+([DVC-files](/doc/user-guide/dvc-file-format)). Each of these stages define
```

Comment:
`defines a single` ?

Reply:
defines a single command

Reply:
> defines a single command

Yes that is more correct syntax, thanks. But I wend for "These stages define individual commands to execute" because I think we want to talk about the commands in plural. See a5019ef.